MCprep_addon/materials/generate.py

```diff
@@ -1334,6 +1334,9 @@ def matgen_cycles_simple(
 	else:
 		principled.inputs["Metallic"].default_value = 0
 
+	# Specular tends to cause some issues with how blocks look, so let's disable it
+	principled.inputs["Specular"].default_value = 0
```

Comment:
DO we think this is true for _all_ blocks? Does this hold for PBR packs? Generally as I understand, the realistic usage of specular of pbr rendering with cycles's principled node is to leave spec at 0.5 (but then roughness drives how rough ofc, so that may be the main issue here).